### PR TITLE
Activate ci

### PR DESCRIPTION
**Pull request by frenchy64**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  java:
+    version: oraclejdk8
+  environment:
+    # Boot sometimes uses over 4G when dowloading deps
+    # Limit could be higher probably, 2-3G?
+    _JAVA_OPTIONS: "-Xms512m -Xmx1024m"
+    BOOT_WATCHERS_DISABLE: "yes"
+dependencies:
+  pre:
+    # Could use `latest` tag instead of version number
```
Comment (dball) [inline circle.yml]: The comment and the code are at odds :)

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+machine:
+  java:
+    version: oraclejdk8
+  environment:
+    # Boot sometimes uses over 4G when dowloading deps
+    # Limit could be higher probably, 2-3G?
+    _JAVA_OPTIONS: "-Xms512m -Xmx1024m"
+    BOOT_WATCHERS_DISABLE: "yes"
```
Comment (calvis) [inline circle.yml]: this only works if the boot version is 2.7.2-SNAPSHOT

File: src/sails_forth/client.clj [hunk 3]
```diff
@@ -6,8 +6,10 @@
             [sails-forth.clojurify :as clj]))
 
 (defprotocol Cache
-  (put! [_ key value])
-  (get! [_ key]))
+  (^:spark/no-boot-spec-coverage
```
Comment (dball) [inline src/sails_forth/client.clj:9]: The only thing I don't like about this metadatum is the specificity. It's a procedural marker - don't do this thing for these vars. How would you feel about, in the next iteration, a declarative marker - `:spark/impure` or similar? One that describes a general property of the fn which the tool could take advantage of?

**Review by dball — APPROVED** [review]
My musing on the metadatum notwithstanding, A++ would approve again

File: src/sails_forth/client.clj (hunk 3 above)
Comment (frenchy64) [reply src/sails_forth/client.clj:9]: I have no issue with changing the name. I was just trying to be hygienic.

**Pull request merged**